libs/net/services/Helpers/ApiService.cs

```diff
@@ -274,10 +274,12 @@ public async Task<IEnumerable<IngestModels.IngestModel>> GetIngestsForTopicAsync
     /// </summary>
     /// <param name="ingest"></param>
     /// <returns></returns>
-    public async Task<IngestModels.IngestModel?> UpdateIngestAsync(IngestModels.IngestModel ingest)
+    public async Task<IngestModels.IngestModel?> UpdateIngestAsync(
```

Comment:
We don't need to edit all the methods to support passing of headers.  You can do all this with dependency injection.  Simply update the `HttpRequestClient` to use dependency injection and then update the startup of each service to set the headers.
However, if you must overload all these functions you need to make the `headers` parameter optional.